app/serializers/work_serializer.rb

```diff
@@ -1,15 +1,15 @@
 class WorkSerializer < ActiveModel::Serializer
```

Comment:
I think you also need to run the serializer through every element found in api/artists_controller#works.

Reply:
(y)